admin/scss/bootstrap/_variables.scss

```diff
@@ -57,7 +57,7 @@ $brand-danger:              #D40404;
 // Control the default styling of most Bootstrap elements by modifying these
 // variables. Mostly focused on spacing.
 
-$spacer:                     1rem;
+$spacer:                     1.2308rem;
```

Comment:
Might be better to write `16/13`? A comment at the top of the section saying "at default zoom there are 13 pixels per rem, so measurements are expressed as desired pixels / rem.

Better yet, make 1/3 a variable called `$px_to_rem` and then say `spacer: (16 * $px_to_rem)rem` or whatever the syntax is.